src/kt_variant.cpp

```diff
@@ -152,6 +159,20 @@ KtVariant::KtVariant(const Variant& variant) {
     TO_KT_VARIANT_FROM[type](value, variant);
 }
 
+KtVariant::KtVariant(const Variant& variant, const KtPropertyInfo* argInfo) {
+    String class_name {argInfo->class_name};
+
+    if (JAVA_USER_TYPE_CONSTRUCTORS.has(class_name)) {
```

Comment:
The user types are registered using the full fqdn. That's why [this change in godot-kotlin-entry-generator](https://github.com/utopia-rise/godot-kotlin-entry-generator/compare/master...totalorder:argument-full-fqdn?expand=1) is needed. I don't know if that would break a lot of other stuff somewhere. Please let me know :)